src/addrman.h

```diff
@@ -716,13 +716,13 @@ friend class CAddrManTest;
     }
 
     //! Return a bunch of addresses, selected at random.
-    std::vector<CAddress> GetAddr(size_t max_addresses, size_t max_pct)
+    std::vector<CAddress> GetAddr(size_t max_addresses, size_t max_pct, Network net = Network::NET_MAX)
```

Comment:
To designate "not specified / any network", maybe `std::optional` would fit better than using the special value `NET_MAX`? I plan to remove `NET_MAX` some day.

Reply:
I was asking myself if `std::optional` would be better here. Thanks!